### DISCUSSION TITLE
Welcome to Study Discussions!

### DISCUSSION THREAD

**Discussion by inyong37** [inyong37/Study#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HongYongGi** [comment]
GOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOD

**Reply by HongYongGi** [reply]
20221226 Checkout~ : )

**Reply by inyong37** [reply]
회사에서 하라는 일은 안하고!

**Reply by HongYongGi** [reply]
This repository is interesting..bb

**Reply by inyong37** [reply]
XD It covers really many fields.

**Reply by HongYongGi** [reply]
Are you  fullstack developer?

**Reply by HongYongGi** [reply]
lolololollolololollolololollolololol

**Reply by inyong37** [reply]
That is the role of a special military service "agent".

**Reply by HongYongGi** [reply]
> That is the role of a special military service "agent".

Ooops

**Reply by HongYongGi** [reply]
일하기 싫다

**Reply by inyong37** [reply]
쏴장님~ 요기에요~

**Reply by HongYongGi** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by HongYongGi** [comment]
Hey~~, I'm here again~~

**Reply by inyong37** [reply]
WALLOO~~~

**Reply by HongYongGi** [reply]
I don't want to work.

I want to go home :)

**Reply by HongYongGi** [reply]
Please send me home.

**Reply by inyong37** [reply]
You can go home whenever you want, but I can't guarantee your life :D

**Reply by HongYongGi** [reply]
I told my boss yesterday that I was quitting. lol

**Reply by inyong37** [reply]
wow for sure?

**Reply by HongYongGi** [reply]
Yeah~~~~

**Reply by HongYongGi** [reply]
I plan to go until the end of January.

**Comment by HongYongGi** [comment]
허허허

**Reply by inyong37** [reply]
Do mang chu

**Reply by HongYongGi** [reply]
돔황챠

**Reply by HongYongGi** [reply]
ㅋㅋㅋㅋㅋㅋㅋ

**Reply by HongYongGi** [reply]
@inyong37 go to home

**Reply by inyong37** [reply]
LGTM

**Reply by HongYongGi** [reply]
Lol :)
